C++/3. Operators.md

```diff
@@ -2,42 +2,38 @@
 
 ## Start
 
-There are certain operators which do addition, and things like that in C++. In this file, I will teach you the basic operators:
+C++ provides certain operators which do addition, and things like that in C++. In this file, I will teach you the basic arithmetic operators:
 
 - `+`
 - `-`
 - `*`
 - `/`
-- `%`
 
 ## What they do
 
-The `+` operator adds things together. \
-The `-` operator subtracts two things. \
-The `*` operator multiplies things together. \
-The `/` operator divides two things. \
-The `%` operator find the reminder of a division.
+The `+` operator adds two numbers. \
+The `-` operator subtracts two numbers. \
+The `*` operator multiplies two numbers. \
+The `/` operator divides two numbers.
 
 ### Code
 
 ```cpp
 #include <iostream>
 
 int main() {
-  int three = 3;
-  int four = 4;
+  double three = 3.0;
+  double four = 4.0;
```

Comment:
Y

Reply:
Is that "Y" a "yes," or "why"?

Reply:
why

Reply:
Wait I understood now (the below comment)

Reply:
Actually, I just mirrored what you already had, it was originally this:
```c++
Int three = 3;
int four = 4;
```

So to use doubles (which have decimal places), it would be:
```c++
double three = 3.0;
double four = 4.0;
```

Reply:
I inlined these in the update, is it better?

Reply:
lemme check

Reply:
ya its better